.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "utils"]
 	path = utils
-	url = https://github.com/goToMain/c-utils.git
+	url = https://github.com/badevos/c-utils.git
```

Comment:
You have to drop this change. You must add a submodule bump commit (after https://github.com/goToMain/c-utils/pull/23 has been merged) instead.